### ISSUE TITLE
BufferedPortImageRgb image read put system on hang problem using Matlab

### ISSUE THREAD

**Issue by smwasif** [robotology/robotology-superbuild#315]
Hi. 
    I have opened `BufferedPortImageRgb` in Matlab using `yarpImage=port.read;`, the system hangs to this `port.read` call and return nothing.  I opened open only once but before read I closed this port to avoid system hang but still I am getting same problem.
Please to resolve this issue.

**Comment by smwasif** [comment]
Duplicate of #
Please help as I need to resolve this issue at priority.

**Comment by traversaro** [comment]
Hi @smwasif , this is more of a generic question, so I will move it to the  https://github.com/robotology/QA repo .
